### ISSUE TITLE
Spelling mistakes xosc/utils.py DynamicConstraints

### ISSUE THREAD

**Issue by dongfeng2134** [pyoscx/scenariogeneration#95]
DynamicsConstraints should be DynamicConstraints in spec
![image](https://user-images.githubusercontent.com/31565799/165218406-6a5a845a-6831-4dca-96b3-bf1ea67fb058.png)

**Comment by mander76** [comment]
Yes, I'm aware.. But it will be a quite big change API wise. Will change when I update to 1.2 and there will be more API changes
